### ISSUE TITLE
Rating question display bug

### ISSUE THREAD

**Issue by traquall** [surveyjs/editor#172]
### Are you requesting a feature, reporting a bug or ask a queation?

Bug

### What is the current behavior?

I have added multiple rating question to my toolbox and all of them display multiple 'box' (1 ,2 ,3 ,4 5) while all my question in the toolbox have only 1 item.
But the json is correct ! It's only a display problem I think

### What is the expected behavior?

To have the same object on my toolbox and on the editor display.

### How would you reproduce the current behavior (if this is a bug)?

Add an item in the toolbox 

```
editor.toolbox.addItem({
		        name: "Module 1",
		        iconName: "icon-bolt",
		        title: "Module 1",
		        json: { "type": "rating", "rateValues": [{value: 0, text: "Bolt"}] } 
		    });
```

and add it in the editor

PS: This occurs only since the last version 0.12.31

Thank you

Traquall

**Comment by andrewtelnov** [comment]
@traquall I have fixed a bug related to rating question couple days ago. As I see you issue has gone away as well: https://plnkr.co/edit/sQ0pTTgDucBjOteiwlrs?p=preview
It works for last released version 0.12.32 that we have released today.

Thank you,
Andrew

**Comment by traquall** [comment]
@andrewtelnov Ok, thank you :)

Traquall
